### PR TITLE
DSD-1433: SimpleGrid in Storybook 7

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1433](https://jira.nypl.org/browse/DSD-1433)

## This PR does the following:

- Updates the `SimpleGrid` Storybook documentation to version 7.

## How has this been tested?

- local build of Storybook

## Accessibility concerns or updates

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Grid/SimpleGrid.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  argTypes: {
+    className: { control: false },
+    columns: {
+      control: { min: 2, type: "number" },
+    },
+    gap: {
+      control: { type: "select" },
+      table: { defaultValue: { summary: "grid.l" } },
+      options: [
+        "grid.xxs",
```
Comment (EdwinGuzman) [inline src/components/Grid/SimpleGrid.stories.tsx]: Update the definition for `GridGaps` to use an `as const` array that can be reused here.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Just one minor update.

File: src/components/Grid/SimpleGrid.stories.tsx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Grid/SimpleGrid.stories.tsx]: Thanks.  As I was working on `Form`, I realized I had forgotten this.

**Comment by bigfishdesign13** [comment]
@EdwinGuzman I will finish up `Form` once this PR is approved and merged.  I need `GridGaps` for the spacing in `Form`.

**Review by EdwinGuzman — APPROVED** [review]

**Comment by EdwinGuzman** [comment]
It would've been okay to have made this change in the next `Form` PR! For next time.

**Pull request merged**